debian/snapcraft-completion

```diff
@@ -26,6 +40,26 @@ _snapcraft()
         COMREPLY=( travis )
         return 0
         ;;
+    --target-arch | --build-for)
+        arch_options="arm64 armhf amd64 riscv64"
```

Comment:
```suggestion
        arch_options="arm64 armhf amd64 i386 ppc64el riscv64 s390x"
```

Reply:
@lengau I did skip them on purpose as in practice cross building to those arches is very unlikely. But that is probably not a good enough excuse to omit those options :)